docs/Reference/API-Methods.md

```diff
@@ -5001,6 +4999,129 @@ specified group of sender and recipients.
       "result": "0x1"
     }
     ```
+### priv_getFilterChanges
+
+Polls the specified filter for a private contract and returns an array of changes that have occurred 
+since the last poll.
+
+Privacy groups do not have blocks and private transactions cannot be pending so unlike 
+[`eth_getFilterChanges`](#eth_getfilterlogs), `priv_getFilterChanges` always returns an array 
+of log objects or an empty list. 
+
+#### Parameters
+
+`data` - 32-byte [privacy Group ID](../Concepts/Privacy/Privacy-Groups.md).
+
+`data` - Filter ID hash
```

Comment:
Drop the hash. It is just an ID :)

Reply:
Done throughout.  This was copied over from the eth filter methods so have fixed there as well.